src/main/java/edu/iis/powp/controllers/ink/LinePlotterInkControlDriver.java

```diff
@@ -3,14 +3,20 @@
 import edu.iis.client.plottermagic.IPlotter;
 import edu.iis.powp.adapter.LineAdapterPlotterDriver;
 
+import java.util.logging.Logger;
+
+import static edu.iis.powp.controllers.ink.CalculateLineLength.calculateLineLength;
+import static java.util.logging.Level.WARNING;
+
 public class LinePlotterInkControlDriver implements IPlotter {
 
     private LineAdapterPlotterDriver plotter;
-    private InkControllerInterface inkController;
+    private SimmulationInkController inkController;
```

Comment:
Why direct coupling with SimmulationInkController when there is an interface InkControllerInterface?